### ISSUE TITLE
Hovering effect to the Record button

### ISSUE THREAD

**Issue by ghost** [Raghwendra-Dey/productivity_meter#32]
As there is no pressing and hovering effect on the Record button(would be introduced after #20 ) yet, so add some hovering effects so that it will look like to be pressed when pressed.

**Comment by Raghwendra-Dey** [comment]
Please edit it to make `Go` name to `Record` which is introduced in #20 . This can only be done after #20 is resolved. So, currently focus on #33 and #27 .

**Comment by Shagufta08** [comment]
hey!
I would like to contribute on this.

**Comment by ghost** [comment]
@Shagufta08 I am working on this as record button is just added by another PR.

**Comment by Shagufta08** [comment]
@urvashi-code1255 
Okay.
No issues!

**Comment by danish1804** [comment]
can i work on this issue @Shagufta08 @Raghwendra-Dey
